pmm-app/src/pmm-qan/panel/components/ManageColumns/ManageColumns.tsx

```diff
@@ -91,6 +100,20 @@ export const ManageColumns = (props) => {
     </div>
   );
 
+  const OptionContent = ({ humanizeName, tooltipText, serviceTypes }) => (
```

Comment:
This should go to a separate file and possibly have an FC<interface>

Reply:
We try to apply a `component per file rule` as much as we can )

Reply:
@tiagomotasantos I suggest to write a unit test for the options (once you land them in a separate file) just to get a sense of how we do unit tests :)

Reply:
Done!